internal/config/config_type.go

```diff
@@ -100,6 +100,21 @@ func (cm *ConfigMap) FindEntry(key string) (ce *ConfigEntry, err error) {
 	return ce, &NotFoundError{errors.New("not found")}
 }
 
+func (cm *ConfigMap) RemoveEntry(key string) {
+	newContent := []*yaml.Node{}
+
+	content := cm.Root.Content
+	for i := 0; i < len(content); i++ {
+		if content[i].Value == key {
+			i++ // skip the next node which is this key's value
```

Comment:
This confused me, but I think it's mostly because iterating over YAML nodes is weird. But from what I can find this is just how it works?

Reply:
sadly yes this is how it works :sweat_smile: